src/ui/Island/Modals/BetaEndModal/ClaimYourNFT.tsx

```diff
@@ -0,0 +1,43 @@
+import React from "react"
+import Button from "shared/components/Interface/Button"
+
+export default function ClaimYourNFT() {
+  return (
+    <>
+      <div
+        className="modal_actions_column column"
+        style={{ alignItems: "end", textAlign: "right" }}
+      >
+        <h2 className="modal_actions_header">Claim your NFT</h2>
+        <p className="modal_actions_description">
+          participate in the beta? See if you are
+          <br />
+          eligible to claim the NFT awards.
+        </p>
+        <Button
+          onClick={(e) => {
+            e.preventDefault()
+            window.open("/", "_blank")
```

Comment:
Can we add a link to external links object? if we don't know specific url let's just use `https://galxe.com/taho` for now